### DISCUSSION TITLE
Assigning amount to Categoriest : Allow a quick way to assign last month amount to the current month

### DISCUSSION THREAD

**Discussion by SylvainMartel** [actualbudget/releases#156] [Feature ideas]
Well, I guess you recognize a "Quality of Life" feature when you lose one and notice it :P 

One such feature that appeared in YNAB was the "Auto-Assign" box.  

Basically, when you clicked on a category, the "Auto-Assign" box would get populated by:

> Assigned last month:  $
> Spent Last month: $
> Average Assigned: $
> Average Spent: $

The first 2 are the one I used monthly, especially `"Assigned last month"`.  When you clicked on that field, it would assign the amount assigned last month to that category to this month category.  A real time saver :) 
The `"Spent last month"` was nice too since it could show a trend where a category wasn't assigned enough money in it.

**Comment by blounsbury-usbr** [comment]
I'm far to lazy to do this by category. There is an option to copy last month's budget to all categories ... I use the 'Copy last month's budget' option at the top of the budget the 3 vertical dots to auto populate my typical budget for all categories. Works perfect for what I need.

![image](https://user-images.githubusercontent.com/1508213/148836154-afa03c28-4d03-4863-a249-d42d7271eedb.png)

**Reply by SylvainMartel** [reply]
Uh, I never noticed those 3 dots... That will help, but I wouldn't want everything to be set like the month before.  I have the bills on the 1st and 15th that are recurrent every month with the same amount(those are the one I'd set automatically with the feature described above), but the rest varies from month to month.
